### ISSUE TITLE
stereo image and depth

### ISSUE THREAD

**Issue by meteorshowers** [bradenhurl/DeepGTAV-PreSIL#7]
Could you tell me how to get the stereo images and depth image?
Thanks a lot

**Comment by bradenhurl** [comment]
Please see issue https://github.com/bradenhurl/DeepGTAV-PreSIL/issues/6

You would be able to set your cameras to be anywhere you would like if this change gets merged. The camera parameters should be set in setRenderingCam.

**Comment by meteorshowers** [comment]
you mean I can set two camera s on a car at the same time?
Thanks a lot!

**Comment by bradenhurl** [comment]
Correct. Although they may be slightly off in time which could make it impossible to do some tasks such as depth completion.
